upgrades/v0.3.0.md

```diff
@@ -9,10 +10,10 @@ This document provides instructions to upgrade from Kroma v0.2.2 to v0.3.0.
 For upgrade, stop your Kroma full node or validator.
 ```bash
 # for full node
-docker compose --profile fullnode down -v
+docker compose --profile fullnode down
```

Comment:
Can I know why the option `-v` removed?

Reply:
As we discussed before, with `-v`, this command will remove the volume also. So we decided to remove the option as in `v1.0.0.md`, but it was not applied in `v0.3.0.md`, so I applied it.